MekHQ/src/mekhq/gui/dialog/ImageChoiceDialog.java

```diff
@@ -799,20 +798,17 @@ public void setText(String text) {
         }
 
         public void setImage(String category, String name) {
-
-            if (null == category
-                    || name.equals(Crew.PORTRAIT_NONE)) {
+            if (category == null || name.equals(Crew.PORTRAIT_NONE)) {
                 return;
             }
-
             // Try to get the image file.
             try {
                 // Translate the root image directory name.
                 if (Crew.ROOT_PORTRAIT.equals(category))
                     category = ""; //$NON-NLS-1$
                 Image image = (Image) items.getItem(category, name);
-                if(null != image) {
-                    if((null != category) && category.startsWith("Pieces/")) {
```

Comment:
category can't be null here (the if above short-circuits the method if it's null)